lnwallet/channel.go

```diff
@@ -1571,7 +1571,8 @@ func (lc *LightningChannel) logUpdateToPayDesc(logUpdate *channeldb.LogUpdate,
 			Amount: lnwire.NewMSatFromSatoshis(
 				btcutil.Amount(wireMsg.FeePerKw),
 			),
-			EntryType: FeeUpdate,
+			EntryType:                FeeUpdate,
+			removeCommitHeightRemote: commitHeight,
```

Comment:
Shouldn't this be add?

Reply:
It should indeed.... Fixed!

Reply:
Thninking a bit more about this, maybe it should really be `remove`.. To make the `feeUpdates` being removed when compacting logs, then the remove height must be set. lmk what you think.

Reply:
Since `FeeUpdates` doesn't really have the same property of getting added and removed at different heights, I decided to just set both fields at the same time.

When compacting logs we'll now properly remove it if the remove height has passed: https://github.com/lightningnetwork/lnd/pull/2397/commits/277949cb0e08095091a6a56c44d34d553a4d02d1

Also added a test to check that it gets removed during log compacting: https://github.com/lightningnetwork/lnd/pull/2397/commits/e6ee835bbe3238365a7e5054f4fd65b2dea05a92

Reply:
Cool yeah it's slightly different since it doesn't have the same add/remove semantics as the others. However, I think we can emulate that based on the two add heights. Nice was going to recommend we extend the tests to ensure that the log is empty after fully processing a fee update, will check out those extra commits!

Reply:
Yeah I thought about this as well, but it makes sense since we want the fee update to be "insta-removed" in a sense, plus there's so secondary message to cleanup the feeupdate add. might it be correct to set both to the same height?